vortexor/src/stake_updater.rs

```diff
@@ -88,11 +88,17 @@ impl StakeUpdater {
                     .collect::<HashMap<Pubkey, u64>>(),
             );
 
-            *last_refresh = Some(Instant::now());
-            shared_staked_nodes
-                .write()
-                .unwrap()
-                .update_stake_map(stake_map);
+            if stake_map.is_empty() {
```

Comment:
in what case would an RPC return an empty stake map? It seems like an RPC bug to return that

Reply:
I was thinking it is really not the business for the validator to do the forwarding especially when we offload the TPU to the vortexor. Clients such as transaction send services are already sending to multiple leaders. But as this already configured by an option, I made change to pass that option to the vortexor receiver as well.